### DISCUSSION TITLE
how to establish files from useStore on new page?

### DISCUSSION THREAD

**Discussion by PhilipZi** [neuefische/questions#81] [Web] [answered]
Guten Morgen zusammen, 
ich tue mich gerade schwer meine generierten Datein aus einer Function (handleSubmit wie im Bild) aus dem useStore in einer anderen Page zu etablieren. Ist dafür immer ein Prozess oder ein Event nötig oder kann ich diese auch Laden, wenn die Seite neu geladen wird? 
Oder kann ich diese Daten direkt nach dem sie geniert werden auf der anderen Page abbilden? Hätte jemand eine passende Doco für mich die mir dabei helfen könnte? 
Ich freu mich über jeden Hilfe! Danke! 
![Screenshot 2022-08-18 at 10 34 20](https://user-images.githubusercontent.com/102238180/185350112-29811254-ea1c-4a22-b047-f80684f7f814.png)
![Screenshot 2022-08-18 at 10 34 41](https://user-images.githubusercontent.com/102238180/185350125-ab27d5ef-71a0-4751-9385-593cfc645ad6.png)

auf diese Datei würde ich gerne die Daten übertragen

![Screenshot 2022-08-18 at 10 39 27](https://user-images.githubusercontent.com/102238180/185350936-b179a7ef-e38d-42ae-bfa8-4c208a68dafa.png)

**Comment by F-Kirchhoff** [comment, marked as answer]
Schau dir mal an wie du in deiner Componente `Timer` dir den `times` State besorgst und versuche dasselbe in deiner `HomePage` zu machen.

**Reply by PhilipZi** [reply]
Vielen Dank! Das hat geholfen! Ich kann die Zeit nun abbilden, allerdings mapped er nun durch alle children und ich möchte eigentlich immer nur den neusten Wert angezeigt bekommen, gibt es da eine Möglichkeit nur das aktuellste child abzubilden und nicht alle vergangenen?
siehe Bilder und vielen Dank, es geht voran!  

![Screenshot 2022-08-18 at 11 45 23](https://user-images.githubusercontent.com/102238180/185365212-fc4ddca1-b6c7-43ef-ae18-a58f99f6fee6.png)

![Screenshot 2022-08-18 at 11 45 49](https://user-images.githubusercontent.com/102238180/185365343-b37040e4-67d5-45cf-8a2f-54a00ba49e24.png)

**Reply by F-Kirchhoff** [reply]
Bitte öffne dafür das nächste Mal eine neue Diskussion, ansonsten finden zukünftige Studis die Fragen nicht.

**Reply by F-Kirchhoff** [reply]
Du musst aus dem Array irgendwie nur den neuesten Wert herausholen. Ist er am Anfang vom Array oder am Ende? Wer weiß? Und dann musst du statt den gemappten Array was anderes rendern. Aber das findest du heraus was das genau sein muss.

**Reply by PhilipZi** [reply]
Habs geschafft! Mega gut dankeschön!!! (length -1)
